projects/swimlane/ngx-ui/src/lib/components/drawer/drawer.component.ts

```diff
@@ -111,23 +126,20 @@ export class DrawerComponent implements OnDestroy {
    *
    * @memberOf DrawerComponent
    */
-  @HostBinding('style.transform')
   transform: string;
 
   /**
    * Drawer width calculation
    *
    * @memberOf DrawerComponent
    */
-  @HostBinding('style.width')
   widthSize: any;
```

Comment:
this as well as `heightSize` could probably be either `string` or `string | number`

Reply:
yeah good call, updated